js/app/src/api_docs/RecordingSnippet.svelte

```diff
@@ -8,7 +8,7 @@
 	export let dependencies: Dependency[];
 	export let short_root: string;
 	export let root: string;
-	export let current_language: "python" | "javascript";
+	export let current_language: "python" | "javascript" | "bash";
```

Comment:
did you forget to finish the code for the API recorder for bash? You never constructed the snippet from the calls like you do for python and js

<img width="910" alt="Screen Shot 2024-06-04 at 4 20 14 PM" src="https://github.com/gradio-app/gradio/assets/9021060/24ad72b6-f4ee-4c09-806e-7a078f60cc72">

Reply:
Actually I haven't implemented it yet, will do after launch. Right now, the API recorder button just does not appear if you select bash. Will double check this and open an issue to implement later

Reply:
https://github.com/gradio-app/gradio/issues/8457